pynumaflow/function/_dtypes.py

```diff
@@ -168,3 +178,29 @@ def __repr__(self):
     def interval_window(self):
         """Returns the interval window for the event."""
         return self._interval_window
+
+
+class Result:
```

Comment:
Is this name predefined for other sdk or numaflow?
Seems a bit too generic to me, maybe `ReduceResult`?

Reply:
fair. will change.